### PR TITLE
Registration enchancements

### PR DESCRIPTION
**Pull request by ybatsiun**
https://trello.com/c/6q7y1vBR/83-registration-form-issues

### CODE REVIEW

File: client_react/src/components/SignUp.js [hunk 1]
```diff
@@ -83,30 +102,34 @@ export default function SignUp({ onRegisterClick, isSignUp, fetchUser, error })
...
-            setErrors(errors => ({ ...errors, passwordError: false }));
-        } else {
-            setErrors(errors => ({ ...errors, passwordError: true }));
-        };
-        handleChange(e);
-    }
+
+        isPasswordsMatch()
+            ? setError(false)
+            : setError(true);
```
Comment (JackKarichkovskiy) [inline client_react/src/components/SignUp.js:130]: You can just `setError(!isPasswordsMatch())`. Right?

File: client_react/src/components/Slider.js [hunk 2]
```diff
@@ -2,7 +2,30 @@ import React, { useState } from 'react';
 import Slider from '@material-ui/core/Slider';
 
 export default function DiscreteSlider({ onChange, value }) {
-    const [localValue, setValue] = useState()
+    const [localValue, setValue] = useState();
+    const marks = [
+        {
+            value: 0,
+            label: 'Begginer',
```
Comment (JackKarichkovskiy) [inline client_react/src/components/Slider.js:9]: Typo: Begginer -> Beginner

File: client_react/src/components/SignUp.js (hunk 1 above)
Comment (ybatsiun) [reply client_react/src/components/SignUp.js]: correct

File: client_react/src/components/Slider.js (hunk 2 above)
Comment (ybatsiun) [reply client_react/src/components/Slider.js]: done

**Pull request merged**